pkg/bpf/prog.go

```diff
@@ -67,3 +74,72 @@ func (t ProgType) String() string {
 
 	return "Unknown"
 }
+
+type attrProg struct {
```

Comment:
Could you add a comment similar to the one you have below, pointing to this?

https://github.com/torvalds/linux/blob/v4.16/include/uapi/linux/bpf.h#L299